packages/react-query/src/shared/hooks/createHooksInternal.tsx

```diff
@@ -312,23 +312,25 @@ export function createHooksInternal<
               const [queryKey, ...rest] = args;
 
               return queryClient.refetchQueries(
-                getArrayQueryKey(queryKey),
+                getArrayQueryKey(queryKey, 'any'),
```

Comment:
refetch currently only has an `any` mode. Without adding a separate type param or separating into two methods we can't tell if we are targeting `query` or `infinite` I don't think?

Reply:
All good